src/app/models/propTypes/list/index.js

```diff
@@ -0,0 +1,34 @@
+import {
```

Comment:
Similar question re: the prop types - can this be simplified at all so future devs don't have the same headaches with the complexity here?

Okay if not though - this is an established pattern - would be nice if we could avoid adding to it if possible

Reply:
I made an effort to simplify the props. Take a look at it and let me know your thoughts

Reply:
I think we need to pay off some of the tech debt in this area soon because it's going to keep costing us a lot of time.

Reply:
What are your thoughts on exporting the props per component like is done in this PR?

Reply:
I wrote an issue to fix this problem https://github.com/bbc/simorgh/issues/4477

Reply:
It's great to know this PR meets the standard defined in that issue

Reply:
The problem with our prop types is that rather than being explicit, they are abstracted and hidden far away. It might be straightforward enough to undo this but I think the real problem is how complex some of the component prop types are, meaning we should really refactor our components API to accept primitive values when possible and not complex objects.

Reply:
Right. By specifying and exporting the prop types, we will leave the complex objects at the lowest level.